modules/cloud-config-container/simple-nva/README.md

```diff
@@ -68,10 +68,12 @@ module "vm" {
 
 | name | description | type | required | default |
```

Comment:
Can we please add some examples in the readme on how to use the module?
Ideally, one for the custom image and commands. One to use the default frr/bgp.

Reply:
Add simple example with BGP configuration (no custom frr.conf provided in this example)

Reply:
can you please add a few lines of description before the BGP example?

Reply:
Added simple description of the setup proposed in the second example, basically an COS based NVA with FRR with a sample file for running BGP session with a neighbor with random IP and ASN.

Reply:
still some minor comments